src/assets/i18n/en.json

```diff
@@ -90,6 +90,7 @@
   "TABLE.MENU": "Menu",
   "TABLE.SELECT_BUTTON": "Option select...",
   "TABLE.EXPORT_BUTTON": "Option export",
+  "TABLE.SHOW_CONFIGURATION_OPTION": "Show/Hide Configuration Option",
```

Comment:
Change text similar to other menu related options. Something like "Option show/hide config."